apps/src/publicKeyCryptography/ModuloClock.jsx

```diff
@@ -56,32 +60,35 @@ const ModuloClock = React.createClass({
     const maximumTimePerSegment = Math.min(1000, maximumDuration / 10);
```

Comment:
Why are we dividing by 10 -- shouldn't this be the modulo value a.k.a. number of slices in the pie?